bindings/wysiwyg-ffi/src/ffi_dom_creation_error.rs

```diff
@@ -51,3 +51,5 @@ impl From<DomCreationError> for wysiwyg::DomCreationError {
         unimplemented!("Error is not needed as input")
     }
 }
+
+impl Error for DomCreationError {}
```

Comment:
Out of curiosity, why is this needed?

Reply:
It won't compile without it. I suspect it's because of the uniffi update.